openlxp_authentication/admin.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib import admin
+from .models import SAMLConf
+
+
+@admin.register(SAMLConf)
+class SAMLConfAdmin(admin.ModelAdmin):
```

Comment:
In general, bias toward full/readable names of objects -- so in this case it would be `SAMLConfigurationAdmin`